gfp_test.go

```diff
@@ -0,0 +1,157 @@
+package bn256
+
+import (
+	"crypto/rand"
+	"encoding/binary"
+	"io"
+	"math/big"
+	"testing"
+)
+
+func randomGF(r io.Reader) *big.Int {
+	k, err := rand.Int(r, p)
```

Comment:
Where is `p` defined?

Reply:
in the constants.go file
https://github.com/cloudflare/bn256/blob/master/constants.go#L16